janus.c

```diff
@@ -108,18 +108,9 @@ static struct janus_json_parameter admin_parameters[] = {
 	{"transaction", JSON_STRING, JANUS_JSON_PARAM_REQUIRED},
 	{"janus", JSON_STRING, JANUS_JSON_PARAM_REQUIRED}
 };
-static struct janus_json_parameter debug_parameters[] = {
```

Comment:
I removed the template because validation happens inside `janus_request_set_boolean_setting` without using `JANUS_VALIDATE_JSON_OBJECT`.  Because I combined different parameters ("debug", "timestamps", "colors") in the same function, one can't use a single template.  My goal was to move duplicate code and the validation for the parameters is the same or very similar.